src/components/product/prod_list/ProdTable.js

```diff
@@ -0,0 +1,39 @@
+import React from "react";
+import { useShopContext } from "../../../store";
+import { BtnAction } from "./BtnAction";
+
+export const ProdTable = () => {
+  const [state] = useShopContext();
```

Comment:
pls, use name matching with the mean of the component.
don't use an abbreviation of the function component's name, except for some situations with easy to understand what the component means.